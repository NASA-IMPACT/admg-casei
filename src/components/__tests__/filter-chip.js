import React from "react"
import renderer from "react-test-renderer"
import FilterChip from "../filter-chip"

const testId = "4c5e9dc3-38ed-40cb-b6ea-cacf96057b53"
const testLabel = "Weather"

describe("FilterChip", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <FilterChip id={testId} label={testLabel} removeFilter={() => {}} />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
