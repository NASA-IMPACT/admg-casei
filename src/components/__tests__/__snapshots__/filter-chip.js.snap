// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterChip renders correctly 1`] = `
<div
  data-cy="filter-chip"
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#303641",
      "borderRadius": "0.2rem",
      "color": "#fff",
      "display": "flex",
      "margin": "0 0.5rem",
      "padding": "0 0.5rem",
    }
  }
>
  <small>
    Weather
  </small>
  <button
    className="button__El-sc-1ets8ks-0 kimGMC button__Button-sc-1ets8ks-1 blDYcX"
    data-cy="remove-filter"
    disabled={false}
    id="close-icon"
    onClick={[Function]}
    radius={null}
    size="small"
    style={
      Object {
        "marginLeft": "0.5rem",
      }
    }
    title="remove filter"
    type="button"
  >
    <span>
      close icon
    </span>
  </button>
</div>
`;
